### PR TITLE
bugfix: menu hover fill the entire header

### LINKED ISSUE #505: styling: menu hover should fill the entire header
The menu item for "Uploads" and "Communities" has a hover when you move the mouse over it. See the light blue area in the screenshot below:

Screenshot 2021-02-02 at 10 22 49

The light blue however should extend to the entire header (the yellow area around the light blue area shown in the screenshot below:

Screenshot 2021-02-02 at 10 35 57

### PR DESCRIPTION
**Pull request by mb-wali**
closes #505

### CODE REVIEW

**Review by zzacharo — DISMISSED** [review]

**Comment by zzacharo** [comment]
@mb-wali can you upload a small screenshot after the fix?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less [hunk 1]
```diff
@@ -27,10 +28,11 @@ nav {
   color: #ccc;
 }
 
-.navbar-options {
-  :hover {
+li.navbar-options:hover{ 
+  background-color: rgba(255, 255, 255, 0.2);
+  a:hover{
```
Comment (zzacharo) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less:33]: Isn't the outer rule enough to extend the background to the whole `li` element?

Comment (mb-wali) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less:33]: apparently not, the [hyperline]() default hover was applying - so i had to add rule of ```transparent```. and text color inside the hyperlink.

**Comment by mb-wali** [comment]
*screenshot*
Screenshot from 2021-02-02 14-28-17

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /*
  *   Copyright (C) 2020 CERN.
+ *   Copyright (C) Graz University of Technology.
```
Comment (zzacharo) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less:3]: You forgot the year here :)

Comment (zzacharo) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less]: I pushed it :)

**Pull request merged**